smack-integration-test/src/main/java/org/jivesoftware/smackx/muc/AbstractMultiUserChatIntegrationTest.java

```diff
@@ -97,6 +97,13 @@ static void createMuc(MultiUserChat muc, String resourceName) throws SmackExcept
         }
     }
 
+    static void createMembersOnlyMuc(MultiUserChat muc, String resourceName) throws SmackException.NoResponseException, XMPPException.XMPPErrorException, InterruptedException, MultiUserChatException.MucAlreadyJoinedException, SmackException.NotConnectedException, MultiUserChatException.MissingMucCreationAcknowledgeException, MultiUserChatException.MucConfigurationNotSupportedException, MultiUserChatException.NotAMucServiceException, XmppStringprepException {
```

Comment:
The type of `resourceName` should be `Resourcepart`. This is also true for the `createModeratedMuc()` method below. It is called everywhere with a resourcepart that is transformed to a string, just to be re-transformed to a resourcepart within the method.

Reply:
Changed in 91897831dd520863fa5adf31565fb73aa836a78e